mdx/api/index.mdx

```diff
@@ -0,0 +1,776 @@
+| makerAssetData        | (Optional) Contains all the identifying information about the asset(s) the order maker is trying to sell.                                                                                                                                                                                                                                         |
+| takerAssetData        | (Optional) Identifying information of the asset(s) the taker must trade in exchange for the maker's asset(s).                                                                                                                                                                                                                                     |
+| makerAssetAmount      | (Optional) Amount of the maker's asset(s) being offered by the maker.                                                                                                                                                                                                                                                                             |
+| takerAssetAmount      | (Optional) Amount of the taker's asset(s) the maker will accept in exchange for their maker asset(s). In order to calculate the price the maker is offering, one can divide the `makerAssetAmount` by the `takerAssetAmount` (the calculation is [a bit more complex](/docs/guides/v3-specification#assetproxy) if multiple assets are involved). |
+| feeRecipientAddress   | (Optional) The entity that will receive any fees stipulated by the order. This is typically used to incentivize off-chain order relay.                                                                                                                                                                                                            |
+| makerFeeAssetData     | (Optional) Identifying token information of the asset(s) the maker must pay to the feeRecipientAddress.                                                                                                                                                                                                                                           |
+| takerFeeAssetData     | (Optional) Identifying token information of the asset(s) the taker must pay to the feeRecipientAddress.                                                                                                                                                                                                                                           |
+| senderAddress         | (Optional) An advanced field that doesn't need to be set. It allows the maker to enforce that the order flow through some additional logic before it can be filled (e.g., a KYC whitelist) -- more on the ability to extend 0x later.                                                                                                             |
+
+
+### Response
```

Comment:
e.g. here's a place where if you control-f straight to the response, you won't see the pagination banner

Reply:
The thing with this is that pagination affects both the request and the response, so I can't just stick it here. Will see how I can improve this though.

Reply:
all good -- it's just one data point so i don't want to over-index on it, and definitely not a ship blocker